modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java

```diff
@@ -537,6 +537,11 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
         if (property.isNullable && !property.required) {
             additionalProperties.put("serdeWith", true);
         }
+
+        // If a property is a base64-encoded byte array, use `serde_with` for deserialization.
+        if (property.isByteArray) {
```

Comment:
Please let me know if you prefer this to be merged with the conditional above.

Reply:
to me both ways are fine at the first glance